gma/src/include/firebase/gma/types.h

```diff
@@ -255,13 +295,29 @@ class AdResult {
   /// information.
   const AdError& ad_error() const;
 
+  /// For debugging and logging purposes, successfully loaded ads provide a
+  /// ResponseInfo object which contains information about the adapter which
+  /// loaded the ad. If the ad failed to load then the object returned from
+  /// this method will have default values. Information about the error
```

Comment:
Hm, maybe.  We basically instruct people to look at this only on AdResult success, so that should be enough. Let me think about adding the is_default() in a different PR.